appstore/README.md

```diff
@@ -1,86 +1,100 @@
 the appstore, users discover and interact with analytic tools and data to explore scientific problems.
 
 ## Management CLI
+
 The appstore management CLI provides uniform commands for using the environment. It provides default values for secrets
-during local development and ways to provide secrets in proudction.
-| Command                          |                   Description                                |
-|:---------------------------------|:-------------------------------------------------------------|
-|bin/appstore tests {product}      | Run automated unit tests with {product} settings.            |
-|bin/appstore run {product}        | Run the appstore using {product} settings.                   |
-|bin/appstore createsuperuser      | Create admin user with environment variable provided values. |
-|bin/appstore image build          | Build the docker image.                                      |
-|bin/appstore image push           | Push the docker image to the repository.                     |
-|bin/appstore image run {product}  | Run automated unit tests with {product} settings.            |
-|bin/appstore help                 | Run automated unit tests with {product} settings.            |
+during local development and ways to provide secrets in production.
+| Command                          | Description                                                  |
+| :------------------------------- | :----------------------------------------------------------- |
+| bin/appstore tests {product}     | Run automated unit tests with {product} settings.            |
+| bin/appstore run {product}       | Run the appstore using {product} settings.                   |
+| bin/appstore createsuperuser     | Create admin user with environment variable provided values. |
+| bin/appstore image build         | Build the docker image.                                      |
+| bin/appstore image frontend      | Pull frontend assets from helx-ui image for local dev.       |
+| bin/appstore image push          | Push the docker image to the repository.                     |
+| bin/appstore image run {product} | Run automated unit tests with {product} settings.            |
+| bin/appstore help                | Run automated unit tests with {product} settings.            |
 
 ## Testing
+
 Automated testing uses the Python standard `unittest` and Django testing frameworks.
 Tests should be fast enough to run conveniently, and maximize coverage. For example, the Django testing framework
 allows for testing URL routes, middleware and other use interface elements in addition to the logic of components.
 
 ## Packaging
+
 Appstore is packaged as a Docker image. It is a non-root container, meaning the user is not a superuser. It packages
-a branch of Tycho cloned within the appstore hierarchy.
+a frontend copied from [helx-ui](https://hub.docker.com/repository/docker/helxplatform/helx-ui).
```

Comment:
I would include the tycho line as well since AppStore now includes the tycho library.

Reply:
My main thought on that is it comes from `pypi` and is `pip` installed so we don't vendor it the same way, but I can add a note back for it.